kyo-reactive-streams/shared/src/main/scala/kyo/interop/flow/package.scala

```diff
@@ -0,0 +1,33 @@
+package kyo.interop
+
+import java.util.concurrent.Flow.*
+import kyo.*
+import kyo.interop.reactivestreams
+import kyo.kernel.Boundary
+import org.reactivestreams.FlowAdapters
+
+package object flow:
```

Comment:
What do you think about exposing these operations as `Stream` extensions in the `kyo` package instead? I'd be nice to keep a companion object as well since people might want to import it in isolation:

```scala
package kyo

object StreamReactiveStreamsExtensions:
     extension [V, S](self: Stream[V, S]):
         // instance methods
     extension (self: Stream.type):
         // companion methods
         
 // export to the kyo package
 export StreamReactiveStreamsExtensions.*
```

Reply:
sounds good, I'' update this

Reply:
now that I though about it, moving this into kyo package would polute it with reactive-streams dependency. I don't think that's a good ideal, right?

Reply:
It'd still be limited to the specific module. `kyo-reactive-streams` needs to be in the classpath for the extension methods to be available

Reply:
I understand what you mean now, will update!